src/main/java/com/google/api/codegen/transformer/py/PythonGapicSurfaceTestTransformer.java

```diff
@@ -174,4 +182,87 @@ private static ModelTypeTable createTypeTable(String packageName) {
     }
     return testCaseViews.build();
   }
+
+  private List<ViewModel> createSmokeTestViews(Model model, GapicProductConfig productConfig) {
+    ImmutableList.Builder<ViewModel> models = ImmutableList.builder();
+    SurfaceNamer surfacePackageNamer = new PythonSurfaceNamer(productConfig.getPackageName());
+    SurfaceNamer testPackageNamer =
+        new PythonSurfaceNamer(surfacePackageNamer.getTestPackageName());
+    for (Interface apiInterface : new InterfaceView().getElementIterable(model)) {
+      ModelTypeTable typeTable = createTypeTable(surfacePackageNamer.getTestPackageName());
+      GapicInterfaceContext context =
```

Comment:
Move this under the if block since it is only needed if a interface has smoke test